.github/workflows/validate_ansible_roles.yaml

```diff
@@ -1,7 +1,7 @@
 name: Validate Ansible Roles
 on:
   push:
-    branches: [ master ]
+    branches: [ '*' ]
```

Comment:
` > Updated GH actions to run on push events for verification.`

I'm not sure what is the intention here. The other branch we have is `stable`
Are you looking to cover releases as well? Maybe `release` the activity type is better suited.

Reply:
I was trying to make sure that when someone is working on this in their forked repo, they get some validation. Obviously, this was only working if you were pushing to the master branch, which no one does unless they're merging. We can chat more about this to see what makes sense.

Reply:
Thanks for clarification.
I don't know what would be the best way to achieve this.
I guess this looks fine to me.... Anyone has other thoughts?